README.md

```diff
@@ -330,120 +477,69 @@ Additionally, you can also locally trigger a deep link to make sure your code is
-  "rich-media-type": "jpg"
+```swift
+extension AppDelegate: UNUserNotificationCenterDelegate {
+    func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
+        let handled = KlaviyoSDK().handle(notificationResponse: response, withCompletionHandler: completionHandler) { url in
+            print("deep link is ", url)
+        }
+        if !handled {
+           // not a klaviyo notification should be handled by other app code
+        }
+    }
 }
 ```
 
-3. A real device's push notification token. This can be printed out to the console from the `didRegisterForRemoteNotificationsWithDeviceToken` method in `AppDelegate`.
-
-Once we have these three things we can then use the push notifications tester and send a local push notification to make sure that everything was set up correctly.
-
+Note that the deep link handler will be called back on the main thread. If you want to handle URI schemes in addition to universal links you implement them as described in [Option1: URI Schemes](#option-1-uri-schemes).
 
-#### Sandbox Support
+## Additional Details
 
-Apple has two environments with push notification support - production and Sandbox. The Production environment supports sending push notifications to real users when an app is published in the App Store or TestFlight. In contrast, Sandbox applications that support push notifications are those signed with iOS Development Certificates, instead of iOS Distribution Certificates. Sandbox acts as a staging environment, allowing you to test your applications in a environment similar to but distinct from production without having to worry about sending messages to real users.
+### Sandbox Support
+Apple has two environments with push notification support - production and Sandbox.
+The Production environment supports sending push notifications to real users when an app is published in the App Store or TestFlight.
+In contrast, Sandbox applications that support push notifications are those signed with iOS Development Certificates, instead of iOS Distribution Certificates.
+Sandbox acts as a staging environment, allowing you to test your applications in a environment similar to but distinct from production without having to worry about sending messages to real users.
 
-Our SDK supports the use of Sandbox for push as well. Klaviyo's SDK will determine and store the environment that your push token belongs to and communicate that to our backend, allowing your tokens to route sends to the correct environments. There is no additional setup needed. As long as you have deployed your application to Sandbox with our SDK employed to transmit push tokens to our backend, the ability to send and receive push on these Sandbox applications should work out-of-the-box.
-#### Testing with Klaviyo
+Our SDK supports the use of Sandbox for push as well.
```

Comment:
We should specify that this is true for v2.2.0 and higher. I think that's the right version, but I'm not 100% sure.

Reply:
Thanks Mark. Updated to add a info block - 
> ℹ️ Sandbox support is available in SDK version 2.2.0 and higher